sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -518,6 +518,24 @@ object SQLConf {
     .bytesConf(ByteUnit.BYTE)
     .createWithDefaultString("10MB")
 
+  val MAX_BROADCAST_TABLE_BYTES =
+    buildConf("spark.sql.maxBroadcastTableBytes")
+      .doc("Configures the maximum size in bytes for a table when performing a broadcast join. " +
+        "The maximum allowed value is 8G.")
```

Comment:
Any particular reason why this value was chosen?

Reply:
In some cases, we need to set a smaller value to avoid using too much memory for broadcast joins. If the actual value of the broadcast table loaded into memory is greater than this value, an exception will be thrown to avoid driver OOM. 8GB is the historical experience value.